samples/charts/data-chart/shape-series/Pages/DataChartRangeSeries.razor

```diff
@@ -0,0 +1,135 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
```

Comment:
this file should be called DataChartShapeSeries.razor instead of DataChartRangeSeries.razor

Reply:
pending